src/types.ts

```diff
@@ -212,7 +216,46 @@ export type TableData = {
   totalRowCount: number;
   totalPages: number;
   paginationNeeded: boolean;
-  rows: Row[];
+  rows: Row[] | undefined;
   columns: Column[];
   totalsPosition: TotalsPosition;
 };
+
+export interface RenderProps {
+  direction?: 'ltr' | 'rtl';
+  selectionsAPI: stardust.ObjectSelections;
+  rootElement?: HTMLElement;
+  layout: TableLayout;
+  changeSortOrder: ChangeSortOrder;
+  rect: stardust.Rect;
+  tableData?: TableData;
+  pageInfo?: PageInfo;
+  setPageInfo?: SetPageInfo;
+  constraints?: stardust.Constraints;
+  translator?: ExtendedTranslator;
+  theme: ExtendedTheme;
+  keyboard?: stardust.Keyboard;
+  footerContainer?: HTMLElement;
+  announce?: Announce;
+  model?: EngineAPI.IGenericObject;
+  manageData?: typeof manageData;
```

Comment:
manageData is a static function, why is it on the renderProps? It also creates a circular dependency importing manageData to this file.

Reply:
Good question. Because in https://github.com/qlik-oss/sn-table/blob/main/src/index.ts#L48, it also calls render. 

RIght, gonna fix that.

Reply:
Done